src/scripts/main.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+// debugger;
+
+const largeImg = document.getElementById('largeImg');
+
+// const srcLargeImg = largeImg.getAttribute('src');
+
+// const linksListItem = document.querySelectorAll('.list-item__link');
+const imgesListItem = document.querySelectorAll('.gallery__img');
+
+for (const img of imgesListItem) {
+  img.addEventListener('click', (e) => {
+    e.preventDefault();
+
+    const srcObject = img.getAttribute('src');
+
+    largeImg.setAttribute('src', srcObject);
+    largeImg.style.height = '348px';
+  });
```

Comment:
no need to add listeners for each image
will be enough to add one listener to gallery_list.